docker-compose/postgres/db_init.sql

```diff
@@ -1244,4 +1244,56 @@ ALTER TABLE localorg_test.default_configs ADD COLUMN autocomplete_function_name
 
 ALTER TABLE localorg_test.default_configs ADD FOREIGN KEY(autocomplete_function_name) REFERENCES localorg_test.functions(function_name);
 
+CREATE TYPE public.http_method AS ENUM (
+    'GET',
+    'PUT',
+    'POST',
+    'DELETE',
+    'PATCH',
+    'HEAD',
+    'OPTIONS',
+    'TRACE',
+    'CONNECT'
+);
+
+CREATE TABLE localorg_test.webhooks (
+    name text PRIMARY KEY,
+    description text NOT NULL,
+    enabled boolean NOT NULL,
+    url text NOT NULL,
+    method public.http_method NOT NULL DEFAULT 'POST',
+    payload_version text NOT NULL,
+    custom_headers json,

```

Comment:
can we make this not null , and add empty {} by default 
as it help in not doing repeated some check in our code